exchangelib/autodiscover.py

```diff
@@ -429,7 +429,7 @@ def _parse_response(response, encoding='utf-8'):
     # There are three possible protocol types: EXCH, EXPR and WEB. EXPR is for EWS. See
     # http://blogs.technet.com/b/exchange/archive/2008/09/26/3406344.aspx
     for protocol in protocols:
-        if get_xml_attr(protocol, '{%s}Type' % RESPONSE_NS) != 'EXPR':
+        if get_xml_attr(protocol, '{%s}Type' % RESPONSE_NS) not in ('EXCH', 'EXPR'):
```

Comment:
Thanks for the explanation! I think we can justify allowing `EXCH` as long as `EXPR` takes precedence. Would that work out for you? I think a config option to `discover()` is overkill for this.